setup.sh

```diff
@@ -20,7 +20,7 @@ osx() {
 
 install_apt() {
     sudo apt-get update || true
-    sudo apt-get -y install git gdb python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg
+    sudo apt-get -y install git gdb python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg nasm golang
```

Comment:
The setup is a basic step which needs to execute for `pwndgb` and the tests for this reason, it is important to install the `nasm` and `go` packages at this time.

Reply:
Solved in this replay: https://github.com/pwndbg/pwndbg/pull/868#discussion_r549089436